pkg/controller/migplan/validation.go

```diff
@@ -105,6 +109,7 @@ const (
 	RegistriesEnsuredMessage              = "The migration registry resources have been created."
 	PvsDiscoveredMessage                  = "The `persistentVolumes` list has been updated with discovered PVs."
 	ClosedMessage                         = "The migration plan is closed."
+	SourcePodsNotHealthyMessage           = "Not healthy pods in some namespaces."
```

Comment:
Suggest: _Source namespaces contain unhealthy pods.  See: \`unhealthyNamespaces\` for details._

Reply:
ready, thanks